packages/protocol/contracts/common/interfaces/IEpochManagerEnabler.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity >=0.5.13 <0.9.0;
+
+interface IEpochManagerEnabler {
```

Comment:
unnecessary to have an interface for this

Reply:
This is used for forge testing and migration scripts.